zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -282,7 +294,8 @@ private void printParagraph(Paragraph paragraph) {
     LOG.info("Paragraph ID : " + paragraph.getId());
     LOG.info("Paragraph title : " + paragraph.getTitle());
   }
-  
+
+  @SuppressWarnings("unused")
```

Comment:
My guess that those procedures were left by the original author as a debug facilities that were useful at some time during the development